types/swagger-schema-official/index.d.ts

```diff
@@ -55,18 +55,19 @@ export interface QueryParameter extends BaseParameter, BaseSchema {
   allowEmptyValue?: boolean;
 }
 
-export interface PathParameter extends BaseParameter {
+export interface PathParameter extends BaseParameter, BaseSchema {
```

Comment:
For proof:

<img width="391" alt="screen shot 2017-10-23 at 3 59 31 pm" src="https://user-images.githubusercontent.com/149178/31872879-3247544e-b80b-11e7-8a6a-dd234f380111.png">